funding/batch_test.go

```diff
@@ -353,10 +353,12 @@ func TestBatchFund(t *testing.T) {
 				tc.failPublish,
 			)
 
+			strategy := lnrpc.CoinSelectionStrategy_STRATEGY_LARGEST
 			req := &lnrpc.BatchOpenChannelRequest{
-				Channels:    tc.channels,
-				SatPerVbyte: 5,
-				MinConfs:    1,
+				Channels:              tc.channels,
+				SatPerVbyte:           5,
+				MinConfs:              1,
+				CoinSelectionStrategy: strategy,
```

Comment:
Because the default behavior now is to fall back to the pre-configured value, we don't need to update all those itests, wich makes the diff even smaller.

Reply:
Thanks this makes total sense since I'm new to protobuf I found we could use the first value in the enum as the default one which in our case would be `STRATEGY_USE_GLOBAL_CONFIG`.